bin/doclint

```diff
@@ -103,6 +103,11 @@ if [[ -z $MODULE_DIR ]]; then
     MODULE_DIR=$ORIG_DIR
 fi
 
+# If the module directory is not an absolute path, make it one
+if [[ $MODULE_DIR != $ORIG ]] && ! [[ -d $MODULE_DIR  ]]; then
+    MODULE_DIR="${ORIG_DIR}/${MODULE_DIR}"
+fi
```

Comment:
`[[ -d $MODULE_DIR ]]` is not a good way to check if the directory is relative - all that tells you is that the directory doesn't exist. Absolute paths to directories which don't exist will also fail that check.

Use `[[ "$MODULE_DIR" = /* ]]` or something similar instead.

Reply:
Updated